bigbluebutton-tests/playwright/learningdashboard/learningdashboard.js

```diff
@@ -0,0 +1,123 @@
+
+    await this.dashboardPage.hasText(e.messageLearningDashboard, '1', ELEMENT_WAIT_LONGER_TIME);
+    
+  }
+
+  async userTimeOnMeeting() {
+    await this.modPage.waitAndClick(e.recordingIndicator);
+    await this.modPage.waitAndClick(e.confirmRecording);
+    await this.modPage.hasText(e.recordingIndicator, '00:0000:00');
+
+    const timeTest = await (this.dashboardPage.getLocator(e.userOnlineTime)).textContent();
+
+    const [hours, minutes, seconds] = timeTest.split(':').map(Number);
+    const timeInSeconds =  hours * 3600 + minutes * 60 + seconds;
+    await sleep(1000);
+    await this.dashboardPage.reloadPage();
+    const timeTestGreater = await (this.dashboardPage.getLocator(e.userOnlineTime)).textContent();
+    
+    const [hoursGreater, minutesGreater, secondsGreater] = timeTestGreater.split(':').map(Number);
+    const timeInSecondsGreater =  hoursGreater * 3600 + minutesGreater * 60 + secondsGreater;
+
+    await expect(timeInSecondsGreater).toBeGreaterThan(timeInSeconds);
+  }
+
+  async polls() {
+    // True/False
+    await openPoll(this.modPage);
+    await this.modPage.type(e.pollQuestionArea, 'True/False?');
+    await this.modPage.waitAndClick(e.pollTrueFalse);
+    await this.modPage.waitAndClick(e.startPoll);
+
+    await this.userPage.waitAndClick(e.pollAnswerOptionBtn);
+    await this.modPage.hasText(e.numberVotes, '1');
+    await this.modPage.waitAndClick(e.cancelPollBtn);
+
+    //ABCD
+    await this.modPage.getLocator(e.pollQuestionArea).fill(' ');
+    await this.modPage.type(e.pollQuestionArea, 'ABCD?');
+    await this.modPage.waitAndClick(e.pollLetterAlternatives);
+    await this.modPage.waitAndClick(e.startPoll);
+    await this.userPage.waitAndClick(e.pollAnswerOptionBtn);
+    await this.modPage.hasText(e.numberVotes, '1');
+    await this.modPage.waitAndClick(e.cancelPollBtn);
+
+    //Yes/No/Abstention
+    await this.modPage.getLocator(e.pollQuestionArea).fill(' ');
+    await this.modPage.type(e.pollQuestionArea, 'Yes/No/Abstention?');
+    await this.modPage.waitAndClick(e.pollYesNoAbstentionBtn);
+    await this.modPage.waitAndClick(e.startPoll);
+    await this.userPage.waitAndClick(e.pollAnswerOptionBtn);
+    await this.modPage.hasText(e.numberVotes, '1');
+    await this.modPage.waitAndClick(e.cancelPollBtn);
+
+    //User Response
+    await this.modPage.getLocator(e.pollQuestionArea).fill(' ');
+    await this.modPage.type(e.pollQuestionArea, 'User response?');
+    await this.modPage.waitAndClick(e.userResponseBtn);
+    await this.modPage.waitAndClick(e.startPoll);
+    await this.userPage.waitForSelector(e.pollingContainer);
+    await this.userPage.type(e.pollAnswerOptionInput, e.answerMessage);
+    await this.userPage.waitAndClick(e.pollSubmitAnswer);
+    await this.modPage.hasText(e.numberVotes, '1');
+    await this.modPage.waitAndClick(e.cancelPollBtn);
+
+    //Checking Polls on Learning Dashboard
+    await this.dashboardPage.waitAndClick(e.pollPanel);
```

Comment:
I think before each beginning of the data check we could forcibly reload the dashboard page and then wait for them to be displayed. Since we have to wait for it, it's easier to avoid failures and excessive timeouts reloading the page